themes/bootstrap3/js/combined-search.js

```diff
@@ -4,15 +4,14 @@ VuFind.combinedSearch = (function CombinedSearch() {
   function init(container, url) {
     VuFind.loadHtml(container, url, '', function containerLoad(responseText) {
       if (!responseText || responseText.length === 0) {
-        if (container.style) {
-          container.style.display = "none";
-        }
-        let parent = container.parentNode;
-        while (parent && parent.classList.contains('js-hide-if-empty')) {
-          if (parent.style) {
+        var element = typeof container === 'string' ? document.querySelector(container) : container;
```

Comment:
From some code hunting and brief testing, I'm not seeing the code path where `container` is *not* a string -- the only place the function is called from is results-ajax.phtml.  Is that correct?  Are we handling the case of it being an element already for some back-compatibility?

Reply:
loadHtml and initResultScripts both support receiving elements directly, so I just did this for consistency with those (and because I imagined that the existing code might have been treating this as an element for some reason). But I don't have any specific knowledge of a scenario where this is not a string... just saw a lot of clues suggesting that such a scenario might exist.

Reply:
Ok, that makes sense then, more future-proofing perhaps.